internal/server/proto/server.proto

```diff
@@ -4018,8 +4044,8 @@ message EntrypointExecRequest {
 
     enum Channel {
       UNKNOWN = 0;
-      STDOUT  = 1;
-      STDERR  = 2;
+      STDOUT = 1;
+      STDERR = 2;
```

Comment:
Random Q: do you have a formatter that does this? I don't mind at all, but it'd be nice if one existed!

Reply:
Yeah my Goland formats this automatically.